benchmark/README.md

```diff
@@ -35,9 +35,38 @@ Source SVG: https://dev.w3.org/SVG/tools/svgweb/samples/svg-files/tiger.svg
 
 |Renderer|Speed|Output|
 |:-------|----:|:----:|
-|svg2png-wasm|20 ops/s, ±0.07%|![svg2png-wasm output](results/tiger-svg2png-wasm.png)|
-|sharp|**21 ops/s, ±2.52%**|![sharp output](results/tiger-sharp.png)|
-|svg2img|11 ops/s, ±4.37%|![svg2img output](results/tiger-svg2img.png)|
+|svg2png-wasm|19 ops/s, ±1.49%|![svg2png-wasm output](results/tiger-svg2png-wasm.png)|
+|resvg-js|**44 ops/s, ±1.05%**|![resvg-js output](results/tiger-resvg-js.png)|
+|resvg|31 ops/s, ±1.62%|![resvg output](results/tiger-resvg.png)|
+|sharp|22 ops/s, ±1.31%|![sharp output](results/tiger-sharp.png)|
 
-_Sun Jan 02 2022 15:24:00 GMT+0000 (Coordinated Universal Time)_
+_Tue Jan 04 2022 12:34:57 GMT+0000 (Coordinated Universal Time)_
+
+
+## text
+
+Source SVG: https://raw.githubusercontent.com/yisibl/resvg-js/main/example/text.svg
+
```

Comment:
The current rendering is incorrect. This SVG needs a Chinese font to render the `<text>` tag.